registration-processor/registration-processor-packet-manager/src/test/java/io/mosip/registration/processor/packet/manager/service/impl/test/IdrepoDraftServiceTest.java

```diff
@@ -223,9 +256,43 @@ public void idrepoDraftReprocessableExceptionTest()
 				Lists.emptyList(), null, IdResponseDTO.class)).thenReturn(idResponseDTO);
 		when(registrationProcessorRestClientService.patchApi(any(), any(), any(), any(), any(), any()))
 				.thenReturn(idResponseDTO1);
+        ResponseDTO discardresponseDTO = new ResponseDTO();
+        discardresponseDTO.setStatus("Drafted");
+        discardresponseDTO.setRegistrationId(ID);
+
+        JSONObject jsonObject1 = new JSONObject();
+        jsonObject1.put("UIN", "1234");
+        discardresponseDTO.setIdentity(jsonObject1);
+        IdResponseDTO discardIdresponceDto=new IdResponseDTO();
+        discardIdresponceDto = new IdResponseDTO();

```

Comment:
Response spell check needed and two objects of IdResponseDTO created removed one ,
one more point Discard api response is it contains status with DRAFTED?

Reply:
changed spelling. IdResponseDTO removed;